lib/check-tags.js

```diff
@@ -0,0 +1,40 @@
+var log = require('./out')({});
+
+function checkTags(app, mod, name) {
+  if ((app.excludeTags && !mod.tags) || (!app.withTags && !app.excludeTags)) {
```

Comment:
The empty array `[]` evaluates to true, so you'll need to use `app.withTags.length` and `app.excludeTags.length` in `if` conditions.

I'm not sure how this would pass the tests.